cmd/pebble/cmd_help.go

```diff
@@ -168,11 +168,15 @@ type helpCategory struct {
 var helpCategories = []helpCategory{{
 	Label:       "Basics",
 	Description: "basic management",
-	Commands:    []string{"run", "autostart", "start", "stop", "exec", "services", "logs"},
+	Commands:    []string{"run", "exec"},
+}, {
+	Label:       "Services",
+	Description: "service management",
+	Commands:    []string{"autostart", "start", "stop", "restart", "services", "logs", "signal"},
```

Comment:
It feels worth reorganizing indeed, but the groups feels a bit blurred, in the sense that it's not clear what goes inside which given their descriptions. Also, as a previous problem we already had, the descriptions are currently not worded evenly: some of them are actions ("view changes") while others are passive ("basic/service management"). Finally, "basic" now has "run" and "exec", which are completely unlike each other, and the proximity of their names doesn't help either.

So maybe we should dedicate a further moment to try to help the situation here and have more distinctive "buckets".

Here is an initial attempt:
```
"Run", "run pebble"
    {"run", "help", "version"}
"Plan", "view and change configuration"
    {"add", "plan", "replan"}
"Services", "manage services"
    {"services", "logs", "start", "restart", "signal", "stop", "replan"}
"Changes", "manage changes and their tasks"
    {"changes", "tasks"}
"Warnings", "manage warnings"
    {"warnings", "okay"}
```
Some notes:
- Isn't "autostart" the old version of "replan"?  That means we can drop the old one.
- If "autostart" and "replan" are indeed friends, it's curious we had them in different groups. What's the right one?
- I've tried to order services inside the groups using some subjective logic while taking into account their meaning, relevance, and proximity

Reply:
The above is missing "exec", which doesn't really fit anywhere. I've updated suggesting adding "Operations" which will have "exec" and (soon) "checks". Let's discuss tonight.

Reply:
For now I've added a "Files" group for files and exec.